build.sh

```diff
@@ -91,7 +91,7 @@ function docker_cmd()
         docker build -t $DOCKERHUB_REGISTRY:${HASH_TAG} .
 
         echo "scanning docker image..."
-        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ aquasec/trivy --exit-code 1 --severity CRITICAL "$DOCKERHUB_REGISTRY":"${HASH_TAG}"
+        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ aquasec/trivy --exit-code 1 --severity CRITICAL --ignorefile .trivyignore "$DOCKERHUB_REGISTRY":"${HASH_TAG}"
```

Comment:
```suggestion
        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ -v "$PWD/.trivyignore":/.trivyignore:ro aquasec/trivy --exit-code 1 --severity CRITICAL --ignorefile /.trivyignore "$DOCKERHUB_REGISTRY":"${HASH_TAG}"
```
this assumes that `docker run` command runs from the repo root - could you confirm?

Reply:
I don't know how jenkins runs these ... will add the volume mount though.

Reply:
Actually, I do think the command runs from the repo root. This is the shell in jenkins:
```
#!/bin/bash
export GIT_COMMIT=$(git log -n 1 --pretty=format:%H)
./jenkins/jenkins_build.sh
```